build.gradle

```diff
@@ -77,6 +77,11 @@ android {
         testInstrumentationRunnerArgument "TEST_USER", "\"$System.env.OCTEST_APP_USERNAME\""
         testInstrumentationRunnerArgument "TEST_PASSWORD", "\"$System.env.OCTEST_APP_PASSWORD\""
         testInstrumentationRunnerArgument "TEST_SERVER_URL", "\"$System.env.OCTEST_SERVER_BASE_URL\""
+
+        jackOptions {
+            enabled true
```

Comment:
Oh, the new Java compiler. This is strictly needed, if build breaks due to it we could remove it (but that's not happenning right now) . ReferencE: https://developer.android.com/preview/setup-sdk.html#get-as13